content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,178 @@ title: Rate Limits
-reported with the error message `too many currently pending authorizations`.
-
-For users of the ACME v2 API you can create a maximum of 300 <a
-id="new-orders"></a>**New Orders** per account per 3 hours. A new order is created each time you request a certificate from the Boulder CA, meaning that one new order is produced in each certificate request. Exceeding the New Orders
-limit is reported with the error message `too many new orders recently`.
-
-# <a id="overrides"></a>Overrides
-
-If you've hit a rate limit, we don't have a way to temporarily reset it. You'll
-need to wait until the rate limit expires after a week. We use a sliding window,
-so if you issued 25 certificates on Monday and 25 more certificates on Friday,
-you'll be able to issue again starting Monday. You can get a list of certificates
-issued for your registered domain by [searching on crt.sh](https://crt.sh), which
-uses the public [Certificate Transparency](https://www.certificate-transparency.org)
-logs.
-
-If you are a large hosting provider or organization working on a Let's Encrypt
-integration, we have a [rate limiting
-form](https://goo.gl/forms/plqRgFVnZbdGhE9n1)
-that can be used to request a higher rate limit. It takes a few weeks to process
-requests, so this form is not suitable if you just need to reset a rate limit
-faster than it resets on its own.
-
-Note that most hosting providers don't need rate limit increases, because
-there's no limit on the number of distinct registered domains for which you can issue.
-So long as most of your customers don't have more than 2,000 subdomains on a
-registered domain, you most likely do not need an increase. See our [Integration Guide](/docs/integration-guide) for more advice.
-
-# <a id="clearing-pending"></a>Clearing Pending Authorizations
-
-If you have a large number of pending authorization objects and are getting a
-Pending Authorizations rate limiting error, you can trigger a validation attempt for those
-authorization objects by submitting a JWS-signed POST to one of its challenges, as
-described in the
-[ACME spec](https://tools.ietf.org/html/rfc8555#section-7.5.1).
-The pending authorization objects are represented by URLs of the form
-`https://acme-v02.api.letsencrypt.org/acme/authz/XYZ`, and should show up in your
-client logs. Note that it doesn't matter whether validation succeeds or fails.
-Either will take the authorization out of 'pending' state. If you do not
-have logs containing the relevant authorization URLs, you need to wait for the
-rate limit to expire. As described above, there is a sliding window, so this may
-take less than a week depending on your pattern of issuance.
-
-Note that having a large number of pending authorizations is generally the
-result of a buggy client. If you're hitting this rate limit frequently you
-should double-check your client code.
+# Foreword
+
+Although Let's Encrypt offers SSL/TLS certificates free of charge, unfortunately
```

Comment:
```suggestion
Although Let's Encrypt offers SSL/TLS certificates free of charge, be aware that
```

Just being real nitpicky here, so feel free to ignore me, but the word "unfortunately" rubbed me the wrong way. I mean, it's a *good* thing that there are HSMs and OCSP servers and audits involved, right? It just seems like just informing people that "there's no such thing as a free lunch" in that even if they're not paying for it, *someone* is, makes more sense to me, rather than it being just "gee it sucks that you're use those resources". But I'm sure a better wordsmith than I could do better.

Reply:
Not a bad suggestion. I chose "unfortunately" to emphasize the negativity of the cost of producing certificates. You seem to have found a very different perspective than I was intending, so I just want to be sure you were reading it correctly first. The intention was to garner empathy for Let's Encrypt to generate respect for the organization and its resources. I'm a little concerned that "be aware that" might come across as foreboding. I walked a bit of a tightrope in trying to balance justification of the policing-nature of this page with welcoming altruism. I'm not sure how well I succeeded though.

Reply:
Yeah, I'm not really thrilled with my wording either. I guess it's just that, as a *user* of the web, it's a good thing that there are so many checks and balances on certificate issuance by a trusted CA, which while it'd be great if that were inherently free, it obviously has *some* costs that *somebody* has to pay. So I was trying to move emphasis from "gee it's a shame that it costs resources to do this service" even more to "it's great that we have sponsors and donors to provide this service to you". Which I guess is what's in the following lines anyway, so perhaps this is just my weird brain that saw something odd about it.